checkout.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: lesson
+subtitle: Checkout Procedure
+email: training-help@software-carpentry.org
+---
+After you have finished the two-day training class,
+you must go through these steps in order to complete your training and be certified:
+
+1.  Submit a [small change to one of our lessons](#lesson-change).
+2.  Take part in an online [discussion session](#discussion-session).
+3.  Teach a short [demonstration lesson](#demo-lesson) online.
+
+We would like to get everyone through this process within two months of training,
+so please get started as soon as you can.
+Anyone who has not completed within 90 days should contact their trainer,
+or the head of instructor training,
+to discuss reasons and options.
+
+## Part 1: Submit a Small Change to One of Our Lessons
+<a name="lesson-change"></a>
+
+Our lessons are maintained and improved by the people who teach them,
+so this part of the checkout procedure gives you a chance to familiarize yourself
+with the way we manage that.
+
+Pick one of the lessons from the list given later in this section
+and submit a *small* change.
+Note that you may submit a change to a Data Carpentry lesson
+when qualifying as a Software Carpentry instructor:
+the goal is to be sure you are familiar with the process.
+
+Your change can be a new exercise or diagram,
+or improve one or more of the lesson's learning objectives.
```

Comment:
The grammar here is "Your change can (be a new...) or (improve one or more...)", but I agree, it'll read more easily as "Your change be X or Y", so I'll do that.